### ISSUE TITLE
Structure for a new section on rheology and material models in ASPECT

### ISSUE THREAD

**Issue by naliboff** [geodynamics/aspect#4908]
I'm currently outlining a new section for the ASPECT manual to describe various rheological approximations and the material structure. Before diving in to far, it would be helpful to get the collective's thoughts on what should be included and organization.

Here is a first stab at an outline, which would start right after the `Choosing a Formulation in ASPECT `section in the [manual](https://aspect-documentation.readthedocs.io/en/latest/user/methods/index.html):

1. Overview of Rheological Approximations in ASPECT
Brief overview of what options there for linear/nonlinear rheology, segway to material model structure
2. Material Model Design
differentiate Rheology versus EOS modules and subsections/submodules therein. These will be referred to in the next section.
3. Distinct Rheology Approximations (refer back 
3.1 Constant Viscosity
3.2 Temperature-dependent viscosity
3.3 Diffusion Creep
3.4 Dislocation Creep
3.5 Peierls Creep
3.6 Simple Composite Viscous Creep
3.7 Diffusion and Dislocation Creep with Grain Size Evolution 
3.8 Plasticity
3.9 Strain-dependent rheology
3.9 Linear Viscoelasticity
3.10 Viscoelastic plastic deformation
3.11 Future/In Progress Work
3.11.1 Iterative Composite Rheology
3.11.2 Coupling rheology to fluid pressure
4. Suggestions for adding new rheology modules and material models

**Comment by cedrict** [comment]
may be a section about invariants ? the two different types (moment and principal), which ones we use, their various formulations, etc ... ?

**Comment by naliboff** [comment]
> may be a section about invariants ? the two different types (moment and principal), which ones we use, their various formulations, etc ... ?

@cedrict  - Definitely, good call!

**Comment by bobmyhill** [comment]
Yes, I think this is good. To summarise my point from our meeting, I think `In Progress Work` is a good section, but maybe we should keep `Future Work` for the GitHub Projects page (https://github.com/geodynamics/aspect/projects).

I'd favour a brief paragraph at the end of each`In Progress` subsection summarising what still needs to be done / how users can use the in progress work (or even whether they should).

To address Anne's point, we should get in touch with the primary authors of potential `In Progress` stuff to ask them if they would be happy to write a short section / give permission for us to write it.

**Comment by tjhei** [comment]
In progress/ future work will likely be outdated quickly, so I agree with putting that into the issue tracker / board instead.

**Comment by gdmate** [comment]
Following on a conversation with a few others in the geodynamics community, there was a request to add a section regarding when/how the prefactor for viscous flow laws should be scaled and include a table of these values.

**Comment by bobmyhill** [comment]
@gdmate We can certainly do that. Do you just mean the scalings from various types of experiments? Would the paragraph from supplementary info S1.3 in Dannberg et al (2017; https://doi.org/10.1002/2017GC006944) be enough or would you like something extra?

**Comment by naliboff** [comment]
@bobmyhill - Sorry, accidentally posted that from my other GitHub account. 

> Do you just mean the scalings from various types of experiments? Would the paragraph from supplementary info S1.3 in Dannberg et al (2017; https://doi.org/10.1002/2017GC006944) be enough or would you like something extra?

Yes, exactly that, +/- a table with the correct values (converted or not) to use from various flow law experiments commonly used in the geodynamics literature.

**Comment by naliboff** [comment]
@bobmyhill @cedrict (and others) - I'm going to try and make headway on this sometime in the next few days. Any other suggestions to the outline in the first post (and subsequent discussion above) that I should take into account before getting started?
